Js/main.js

```diff
@@ -1,17 +1,4 @@
 
-//------------------------ Adding Menu list-------------------------
-
-// const ul =document.querySelector('.menu-nav ul');
-// const li=document.createElement('a');
-// const Profile =document.createElement('a');
-
-// ul.append(li);
-// ul.append(Profile);
-// Profile.innerText=('Profile'); 
-// Profile.classList.add('menu-content-list');
-// li.innerText=('contact'); 
-// li.classList.add('menu-content-list');
-
 
 
```

Comment:
- [ ] You are not using correct linters setup because i don't see linter for javascript which is ESLint. Kindly add the correct setup of linters using this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) 👍